src/test/java/com/aws/greengrass/ipc/IPCEventStreamServiceTest.java

```diff
@@ -75,35 +69,19 @@
     private Topic mockRelativePath;
 
     @Mock
-    private GreengrassCoreIPCService greengrassCoreIPCService;
-    @Mock
-    private software.amazon.awssdk.eventstreamrpc.AuthenticationHandler mockAuthenticationHandler;
-    @Mock
-    private AuthorizationHandler mockAuthorizationHandler;
+    private AuthenticationHandler mockAuthenticationHandler;
 
     @BeforeEach
-    public void setup() {
-        AuthenticationData authenticationData = new AuthenticationData() {
-            @Override
-            public String getIdentityLabel() {
-                return "EventStreamConnectionTest";
-            }
-        };
-        when(mockAuthenticationHandler.apply(any(), any())).thenReturn(authenticationData);
-        when(mockAuthorizationHandler.apply(eq(authenticationData))).thenReturn(Authorization.ACCEPT);
-        when(greengrassCoreIPCService.getAuthenticationHandler()).thenReturn(mockAuthenticationHandler);
-        when(greengrassCoreIPCService.getAuthorizationHandler()).thenReturn(mockAuthorizationHandler);
-        when(greengrassCoreIPCService.getServiceModel()).thenReturn(GreengrassCoreIPCServiceModel.getInstance());
-
-        ipcEventStreamService = new IPCEventStreamService(mockKernel, greengrassCoreIPCService, config);
-        NucleusPaths nucleusPaths = mock(NucleusPaths.class);
+    public void setup() throws UnauthenticatedException {
```

Comment:
I updated the test to setup the mock according to how `IPCEventStreamService` is currently implemented.